README.md

```diff
@@ -15,7 +15,7 @@ And that's it!
 
 ## Deploy via One-Click
 
-[![Deploy to Scalingo](https://cdn.scalingo.com/deploy/button.svg)](https://my.scalingo.com/deploy)
+[![Deploy to Scalingo](https://cdn.scalingo.com/deploy/button.svg)](https://my.scalingo.com/deploy?source=https://github.com/Scalingo/sample-node-express#master)
```

Comment:
Can you replace the old link `my.scalingo.com` by `dashboard.scalingo.com` like this
```suggestion
[![Deploy to Scalingo](https://cdn.scalingo.com/deploy/button.svg)](https://dashboard.scalingo.com/create/app?source=https://github.com/Scalingo/sample-node-express#master)
```

Reply:
It is documented here: https://doc.scalingo.com/platform/deployment/one-click-deploy#display-a-one-click-deploy-button

Reply:
Sure, however, it is documented otherwise here: https://developers.scalingo.com/one-click/index

Which is preferred?

Our documentation is inconsistent.

Reply:
The main documentation, the one i shared. The one on developers, is not up-to-date